### PR TITLE
Rs 2023.05.20.1

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Arcueid/Day 4/04_00_ARC04_1.txt [hunk 1]
```diff
@@ -127,17 +132,18 @@ Looking closely, I notice my school uniform in her hand.
...
 -- Page 12, Offset 5594.
 -- 「……はい。重ねて申し訳ありません。
-"Yes. Once again, I apologize.
+"Yes. Once again, I must apologize.
 }
 [sha:b3d651a3c00432bbc88c78c2d0187e8c2fb25d63]{
 -- Page 12, Offset 5595.
 -- お見苦しいところをお見せしました」
-For making you see something so unpleasant."
+I caused you to see something unpleasant."
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_1.txt:140]: oh cmon
>It seems I have embarrassed myself in front of you.

something like this. this phrase is uttered when you commit a faux pas or whatever

5598
>I don't think she did anything particularly embarrassing, but...

File: script/Arcueid/Day 4/04_00_ARC04_1.txt [hunk 2]
```diff
@@ -287,7 +297,7 @@ The touch of the washcloth is enough to send a stinging pain through my throat.
 [sha:33e04eebd2591bbb4d3fef62fb2d955fb9930d89]{
 -- Page 24, Offset 5625.
 -- 「……なんだ、これ」
-"What is this...?"
+"What the hell...?"
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_1.txt:300]: not more of them nooooooo
>What's... %{i}that%{/i}?

File: script/Arcueid/Day 4/04_00_ARC04_1_1.txt [hunk 3]
```diff
@@ -1,13 +1,13 @@
 [sha:cc509395dec627bbbd6ed7e85c207214be94bbec]{
 -- Page 10003, Offset 5731.
 -- 「そういえば。思い出したんだけど、秋葉は今も浅女の生徒なんだよな……？」
-"Oh, by the way.
- If I remember correctly, you still attend Asagami Private Girl's Academy, don't you...?"
+"Oh, that's right――
+if I remember correctly, you still attend Asagami Private Girl's Academy, don't you?"
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_1_1.txt:5]: feel like just saying "Asagami" here is totally fine.
You don't say the full names of school generally speaking

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 4]
```diff
@@ -143,7 +143,7 @@ And yet, through the gaps between the cars racing by,
 [sha:357915016fe7a96bfcb302ba91f84eb7b09e2386]{
 -- Page 110, Offset 5837.
 -- そこにいたのは、彼女だった。
-Over there is a woman.
+There's a woman over there.
 }
 [sha:72441e21d1a103bfcdede377a26019dda0d65bde]{
 -- Page 110, Offset 5838.
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:149]: 5839
>Golden hair reminiscent of sunlight.

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 5]
```diff
@@ -143,7 +143,7 @@ And yet, through the gaps between the cars racing by,
 [sha:357915016fe7a96bfcb302ba91f84eb7b09e2386]{
 -- Page 110, Offset 5837.
 -- そこにいたのは、彼女だった。
-Over there is a woman.
+There's a woman over there.
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:146]: >And there she is.

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 6]
```diff
@@ -404,7 +404,7 @@ could possibly still be alive.
 [sha:647a529931746fbbfffd3dd8461a080d472e4660]{
 -- Page 122, Offset 5887.
 -- なのに、生きている女は、俺を見て笑った。
-And yet, that woman is alive, and smiling at me.
+And yet, that woman %{i}is%{/i} alive, and smiling at me.
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:407]: >And yet, she is %{i}is%{/i} alive, and smiling at me.

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 7]
```diff
@@ -1031,17 +1033,18 @@ It was a reply I didn't want to hear,
 [sha:792c05fab68ff1bfe830661b5d761104fd05590b]{
 -- Page 159, Offset 6008.
 -- 「ふざけるな、死んだ人間が生きてるハズがないだろう！」
-"―――Don't be ridiculous! A person who's dead can't possibly be alive!"
+"Bullshit you are!
+ A person who's dead can't possibly be alive!"
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:1037]: Hard to make it sound non-silly, I suppose.

>...
>How could someone dead possibly be alive?!

File: script/Arcueid/Day 4/04_00_ARC04_3.txt [hunk 8]
```diff
@@ -1113,12 +1116,13 @@ There's no denying that's what the woman in front of me said.
...
 -- 「……人間じゃ、ない……？」
-"You're not... human...?"
+"You're... not human?"
 }
 [sha:83337f93019297eabf0a5c3ee43d09a0f7a8a5d5]{
 -- Page 165, Offset 6025.
 -- 「当たり前でしょ。手足をバラバラにされて、ひとりでに再生できる人間がいると思う？」
-"Of course not. Do you think there are people who can regenerate themselves after being torn limb from limb?"
+"Of course not.
+ Do you seriously believe there are people who can regenerate themselves after being torn limb from limb?"
```
Comment (Petrikow) [inline script/Arcueid/Day 4/04_00_ARC04_3.txt:1125]: people > humans
to emphasize the inhumanity

File: script/Arcueid/Day 4/04_00_ARC04_1.txt (hunk 2 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 4/04_00_ARC04_1.txt:300]: It's a handy phrase! Suggestion kinda too strong / makes it sound like he's already spotted something when it comes before him actually looking in the mirror. Reads to me as more of a "why did my neck hurt" than a "what's that redness on my neck" comment.

Comment (Petrikow) [reply script/Arcueid/Day 4/04_00_ARC04_1.txt:300]: Maybe like

>That's... weird.

or something then

**Pull request merged**